Readability.js

```diff
@@ -143,7 +143,7 @@ Readability.prototype = {
 
   UNLIKELY_ROLES: [ "menu", "menubar", "complementary", "navigation", "alert", "alertdialog", "dialog" ],
 
-  DIV_TO_P_ELEMS: [ "A", "BLOCKQUOTE", "DL", "DIV", "IMG", "OL", "P", "PRE", "TABLE", "UL", "SELECT" ],
+  DIV_TO_P_ELEMS: [ "BLOCKQUOTE", "DL", "DIV", "IMG", "OL", "P", "PRE", "TABLE", "UL" ],
```

Comment:
It looks like the addition of `select` was from https://bugzilla.mozilla.org/show_bug.cgi?id=785343 - I bet all the pages cited there have changed and this was before we added unit-y tests for this kind of thing (in fact, it was before I was involved with this code!) so hard to know what to do about it now. Do we perhaps scrap `<option>` elements elsewhere, to avoid counting them as 'text' for the scoring algorithm?

Reply:
>  Do we perhaps scrap <option> elements elsewhere, to avoid counting them as 'text' for the scoring algorithm?
I will add this in the next commit. I'd also like to change `DIV_TO_P_ELEMS` to a Set since it is only used to test for membership and I think a Set is better for this. Is that okay?

Reply:
Yep, using a `Set` sounds great, thank you!